_topic_maps/_topic_map.yml

```diff
@@ -2949,14 +2949,21 @@ Topics:
   File: what-huge-pages-do-and-how-they-are-consumed-by-apps
   Distros: openshift-origin,openshift-enterprise
 - Name: Low latency tuning
-  File: cnf-low-latency-tuning
+  Dir: low_latency_tuning
   Distros: openshift-origin,openshift-enterprise
-- Name: Performing latency tests for platform verification
-  File: cnf-performing-platform-verification-latency-tests
+  Topics:
+  - Name: Understanding low latency
+    File: cnf-understanding-low-latency
+  - Name: Tuning nodes for low latency with the performance profile
+    File: cnf-tuning-low-latency-nodes-with-perf-profile
+  - Name: Provisioning real-time and low latency workloads
+    File: cnf-provisioning-low-latency-workloads
+  - Name: Debugging low latency tuning
+    File: cnf-debugging-low-latency-tuning-status
+  - Name: Performing latency tests for platform verification
+    File: cnf-performing-platform-verification-latency-tests
```

Comment:
This assembly/module is harder to locate on this PR. I cannot see it's sections listed either. Not sure why this is.

Reply:
This assembly is untouched in this PR.